components/match2/commons/match_summary_base.lua

```diff
@@ -184,6 +185,22 @@ function Footer:addLinks(links)
 		end
 	end
 
+	Array.forEach(MATCH_LINK_PRIORITY, function(linkTypePrefix)
+		for linkType, link in Table.iter.pairsByPrefix(links, linkTypePrefix, {requireIndex=false}) do
+			processLink(linkType, link)
+		end
+	end)
+
+	local unorderedLinks = Table.filterByKey(links, function(key)
+		return Array.all(MATCH_LINK_PRIORITY, function (element)
+			return not String.startsWith(key, element)
+		end)
+	end)
```

Comment:
```suggestion
		local processedKeys = {}
	Array.forEach(MATCH_LINK_PRIORITY, function(linkTypePrefix)
		for linkType, link in Table.iter.pairsByPrefix(links, linkTypePrefix, {requireIndex=false}) do
			processLink(linkType, link)
			table.insert(processedKeys, linkType)
		end
	end)

	local unorderedLinks = Table.filterByKey(links, function(key)
		return Array.find(processedKeys, FnUtil.curry(Operator.eq, key)) ~= nil
	end)
```
@hjpalpha what do you think of this to solve ambiguities?

Reply:
hmm
how about copying the links table (coming from the match2 data) and using the copy to during the display building
that way you can just nil the ones that were already done during the prio group stuff